app/models/registration.js

```diff
@@ -0,0 +1,85 @@
+import DS from 'ember-data';
+import Validator from '../mixins/model-validator';
+
+export default DS.Model.extend(Validator, {
+    contactName: DS.attr('string'),
```

Comment:
If we are requiring users to be logged in could we just use the logged in user's contact information?